docs/manage-data.md

```diff
@@ -0,0 +1,30 @@
+# Data management
+
+eoAPI-k8s provides a basic data ingestion process that consist of manual operations on the components of the stack.
+
+# Load data
+
+## Preshipped bash script
+
+Execute `make ingest` to load data into the eoAPI service.
```

Comment:
I think this should explicitly call out that the `make ingest` target _expects_ two files at the root of the repo. (To save having to dive into a bash script/errors to find that out

Reply:
Yes, indeed, thanks for raising this. I adjusted the docs.